config-builder/src/test/java/com/ca/apim/gateway/cagatewayconfig/BundleFileBuilderTest.java

```diff
@@ -56,9 +56,9 @@ void beforeEach() {
     @Test
     void buildBundleNoSource() {
         BundleFileBuilder bundleFileBuilder = new BundleFileBuilder(documentTools, documentFileUtils, entityLoaderRegistry, bundleEntityBuilder, bundleCache);
-        bundleFileBuilder.buildBundle(null, new File("output"), Collections.emptyList(), "my-bundle");
+        bundleFileBuilder.buildBundle(null, new File("output"), Collections.emptyList(), "my-bundle", "1.0");
 
-        verify(bundleEntityBuilder).build(argThat(bundle -> bundle.getPolicies().isEmpty()), eq(EntityBuilder.BundleType.DEPLOYMENT), any());
+        verify(bundleEntityBuilder).build(argThat(bundle -> bundle.getPolicies().isEmpty()), eq(EntityBuilder.BundleType.DEPLOYMENT), any(), "my-bundle", "1.0");
```

Comment:
Is this test passing? I guess it should contain `eq("my-bundle)` and `eq("1.0")`
Same for other JUnits as well.